cassandra_check/check.py

```diff
@@ -0,0 +1,78 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import re
+import shlex
+
+# project
+from checks import AgentCheck
+from utils.subprocess_output import get_subprocess_output
+from collections import defaultdict
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'cassandra_check'
+DEFAULT_HOST = 'localhost'
+DEFAULT_PORT = '7199'
+
+class CassandraCheck(AgentCheck):
+
+    datacenter_name_re = re.compile('^Datacenter: (.*)')
+    host_status_re = re.compile('^(?P<status>[UD])[NLJM].* (?P<owns>(\d+\.\d+%)|\?).*')
```

Comment:
Should we really send the state as a tag ? Appart from the state `normal`, these are temporary states.
For the `status` should this be a gauge (1 for up, 0 for down) ? or a service check ? both ? In the check they wrote, TLP sent a gauge so they could use it to fill a host map and see down host at a glance, so I can see the use of sending a gauge here.